lib/target/aquarius/classic/aquarius_crt0.asm

```diff
@@ -76,7 +85,15 @@ l_dcal:
     INCLUDE	"crt/classic/crt_section.inc"
 
     SECTION code_crt_init
+IF CLIB_AQUARIUS_PLUS = 1
+    ; Remap the border color character for aqplus
+    in      a, (IO_VCTRL)
+    or      VCTRL_REMAP_BC
```

Comment:
Should 40 column mode be forced here so we have a known starting point for the video hardware?